README.md

```diff
@@ -7,13 +7,9 @@ This plugin is a Notion like database based on folders.
 - **[Documentation](https://rafaelgb.github.io/obsidian-db-folder/)**
 - **[Roadmap](https://github.com/RafaelGB/obsidian-db-folder/projects/1)**
 - **[changelog](https://rafaelgb.github.io/obsidian-db-folder/changelog/)**
-### How can I add a database?
-You can add a folder database by right clicking on the folder where you want to store your table
-
-<img src="https://github.com/RafaelGB/obsidian-db-folder/blob/master/docs/resources/AddDatabase.gif" width="200" height="250"/>

```

Comment:
Why did you remove all the media resources?

Reply:
Because It only shows the right-click folder method of creating the database. So the user will think that is the only way. I probably should've added more ways instead of removing it entirely, you're probably right.

Reply:
Ok, so new media could be added with future PRs

Reply:
Yes, I was thinking when the 0.16 is out, I could add image/videos for README and for the docs.